package.json

```diff
@@ -85,15 +85,16 @@
     "core:test:watch": "ng test clr-core --watch",
     "core:test:unit": "ng test clr-core --watch=false",
     "core:test:treeshaking": "webpack --config src/clr-core/test-bundles/webpack.config.js && bundlesize --config src/clr-core/test-bundles/bundlesize.json",
-    "core:build": "npm-run-all core:build:tokens core:build:sass -p 'core:build:ts -- --force' core:build:api -s core:build:package core:build:clean",
+    "core:test:bundles": "source-map-explorer dist/test-bundles/webpack.bundle.js",
+    "core:build": "npm-run-all core:build:tokens core:build:sass 'core:build:ts -- --force' core:build:sass:min -p core:build:api -s core:build:package core:build:clean",
```

Comment:
The sass minification step must be after the TS builds now as the CSS is written into the JS output. After the TS is built we  use purge to "treeshake" the common css shared between components from the js file.